tachyon/crypto/commitments/polynomial_openings.h

```diff
@@ -237,7 +270,8 @@ class PolynomialOpeningGrouper {
       PolyOracleDeepRef poly_oracle, PointDeepRef point) {
     auto it = std::find_if(
         poly_openings.begin(), poly_openings.end(),
-        [poly_oracle, point](const PolynomialOpening<Poly, PolyOracle>& poly_opening) {
+        [poly_oracle,
+         point](const PolynomialOpening<Poly, PolyOracle>& poly_opening) {
```

Comment:
It is weird. It seems like the formatter is malfunctioning.

Reply:
Formatting was not done in the previous commit. I'll rebase it.